developer-process/directory-structure.md

```diff
@@ -0,0 +1,81 @@
+# Directory Structure
+
+This file describes the model used for code management.
+
+Most vis.gl repositories are monorepos, meaning that multiple submodules are published to npm from the same repo with synchronized version numbers.
+
+## Monorepo directory structure (Legacy)
+
+This is the legacy directory structure used by some projects. New projects should adopt the new directory structure as described in the next section.
```

Comment:
Describe the new structure first, then the legacy structure (unless strong reasons to keep it, just delete the legacy structure).